packages/schemas/tables/sign_in_methods.sql

```diff
@@ -0,0 +1,11 @@
+create table sign_in_methods (
+  id varchar(128) not null,
+  name varchar(128) not null,
+  connector_id varchar(128),
+  metadata jsonb /* @use SignInMethodMetadata */,
+  primary key (id),
+  constraint fk_connector
```

Comment:
will it be better to follow the naming convention as indices? please refer to other sql files.

Reply:
updated